plugins/github-actions/api-report.md

```diff
@@ -142,6 +174,431 @@ export class GithubActionsClient implements GithubActionsApi {
     RestEndpointMethodTypes['actions']['getWorkflowRun']['response']['data']
   >;
   // (undocumented)
+  listBranches(options: {
+    hostname?: string;
+    owner: string;
+    repo: string;
+    page?: number;
+  }): Promise<
```

Comment:
The return results seem very verbose and I'm not sure we are using all of this, can we trim this down to only what is needed?